site/en/blog/longer-esw-lifetimes/index.md

```diff
@@ -37,7 +38,7 @@ Unfortunately, this behavior did not apply to extension events. Extension events
 
 As of Chrome 110, all events reset the lidle timer and the idle timeout will not occur if there are pending events. In other words, assuming there are no unexpected interruptions, extension service workers will now stay alive as long as they are actively processing events.
 
-While this change removes the five minute limit on extension service worker lifetimes, you are still strongly encouraged to design your extensions to be event-based and guard against unexpected termination. Not only should you refrain from keeping your extensions alive indefinitely (which we consider an anti-pattern), you should test your extensions to ensure that they're not doing this unintentionally.
+Because the service worker will still terminate after five minutes if there are no events, you are still strongly encouraged to design your extensions to be event-based and guard against unexpected termination. Not only should you refrain from keeping your extensions alive indefinitely (which we consider an anti-pattern), you should test your extensions to ensure that they're not doing this unintentionally.
```

Comment:
This still isn't quite right.
- The service worker will terminate after 30 seconds if there are no events
- The service worker will terminate after 5 minutes if a single request (event or API call) takes longer than 5 minutes to process

Reply:
PTAL